pkgs/development/python-modules/catboost/format.patch

```diff
@@ -0,0 +1,15 @@
+Don't use variable format string, which triggers -Wformat-security.
```

Comment:
Can you create a upstream PR and use fetchpatch?

Reply:
PR: https://github.com/catboost/catboost/pull/1528
I'll look into using `fetchpatch`.